src/plugins/calculationTry1/handlersCountCellsData.ts

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { MathCounter } from './MathService';
```

Comment:
before this line u can add an empty line (to divide libraries and project things.